### PR TITLE
feat: migrate word

### PR DESCRIPTION
**Pull request by MohdImran001**
Part of #72.
Test with `npm run build && npx mocha test/word.unit.js`

### CODE REVIEW

File: src/word.ts [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+      this[name] = this[name].bind(this);
+    }
+  }
+
+  /**
+   * Returns an adjective of random or optionally specified length.
+   * If specified length is unresolvable, returns random adjective.
+   *
+   * @method faker.word.adjective
+   * @param {number} [length] - optional length of word to return
```
Comment (Shinigami92) [inline src/word.ts:19]: Please remove the JSDoc types

like 
```suggestion
   * @param optional length of word to return
```

any many more below

File: src/word.ts [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+   * @returns {string}          a random adjective
+   */
+  adjective(length?: number): string {
+    var wordList = this.faker.definitions.word.adjective;
+    if (length) {
+      wordList = this.faker.definitions.word.adjective.filter(function (
```
Comment (Shinigami92) [inline src/word.ts:25]: convert to arrow functions to not mess with `this`

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

File: src/word.ts (hunk 2 above)
Comment (MohdImran001) [reply src/word.ts:25]: are you talking about `filter` function ?

Comment (Shinigami92) [reply src/word.ts:25]: yes :)

**Review by prisis — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**